### ISSUE TITLE
Error since SvelteKit 2 and "moduleResolution": "bundler"

### ISSUE THREAD

**Issue by bfovez** [lucide-icons/lucide#1776]
### Package

- [ ] lucide
- [ ] lucide-angular
- [ ] lucide-flutter
- [ ] lucide-preact
- [ ] lucide-react
- [ ] lucide-react-native
- [ ] lucide-solid
- [X] lucide-svelte
- [ ] lucide-vue
- [ ] lucide-vue-next
- [ ] Figma plugin
- [ ] source/main
- [ ] other/not relevant

### Version

0.303.0

### Browser

- [ ] Chrome/Chromium
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] iOS Safari
- [ ] Opera
- [X] Other/not relevant

### Operating system

- [ ] Windows
- [ ] Linux
- [ ] macOS
- [X] Other/not relevant

### Description

Since Sveltekit 2, all new apps are created with this default in ./.svelte-kit/tsconfig.json:

```
{
	"compilerOptions": {
		...
		"moduleResolution": "bundler",
		...
	},
	...
}
```
Since, this kind of import breaks:
```
import type { IconProps } from 'lucide-svelte/dist/types'
Cannot find module 'lucide-svelte/dist/types' or its corresponding type declarations.ts(2307)
```

A workaround is to force the `moduleResolution `option to `node`:

```
{
	"extends": "./.svelte-kit/tsconfig.json",
	"compilerOptions": {
		...
		// Force this the because of lucide-dev
		"moduleResolution": "node"
		...
	}
...
}
```

### Steps to reproduce

1. npm create svelte@latest my-app (just a skeleton project, with typescript)
2. cd my-app
3. npm i lucide-svelte@latest
4. Create the file /src/routes/demo.ts and paste:
`import type { IconProps } from 'lucide-svelte/dist/types'`
5. See the error:
`Cannot find module 'lucide-svelte/dist/types' or its corresponding type declarations.ts(2307)`
Here the whole project that just needs to be "npm i && npm run dev".
[my-app.zip](https://github.com/lucide-icons/lucide/files/13818848/my-app.zip)

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)

**Comment by JClackett** [comment]
Seems like the types have been removed from exports?
any reason these arent being exported anymore?

```
``

**Comment by ericfennis** [comment]
@JClackett You showing types from `lucide-react`.  
This about `lucide-svelte`

**Comment by JClackett** [comment]
Indeed, but since this version there are no exported types from react native package, so was wondering if it was connected

**Comment by ericfennis** [comment]
No probably not `lucide-react-native` had an old types build script because a lot packages where still javascript files instead of typescript. We now have everything in typescript, including `lucide-svelte`.

**Comment by ericfennis** [comment]
Created a fix in #1779. Not all types where exported 🤷🏼‍♂️. 

The new correct syntax for import types will be:
```ts
import type { IconProps } from 'lucide-svelte'
```

**Comment by JClackett** [comment]
Where do I import the types from in a react native project?
